tools/openms/ClusterMassTraces.xml

```diff
@@ -41,16 +41,16 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
     <configfile name="hardcoded_json"><![CDATA[{"log": "log.txt", "threads": "\${GALAXY_SLOTS:-1}", "no_progress": true}]]></configfile>
   </configfiles>
   <inputs>
-    <param name="in" argument="-in" type="data" format="consensusxml" optional="false" label="Mass traces" help=" select consensusxml data sets(s)"/>
-    <param name="min_pearson_correlation" argument="-min_pearson_correlation" type="float" optional="false" value="0.7" label="Minimal pearson correlation score" help=""/>
-    <param name="min_peak_nr" argument="-min_peak_nr" type="integer" optional="false" value="1" label="Minimal peak nr to output pseudo spectra" help=""/>
-    <param name="max_lag" argument="-max_lag" type="integer" optional="false" value="1" label="Maximal lag" help=""/>
-    <param name="max_rt_apex_difference" argument="-max_rt_apex_difference" type="float" optional="false" value="5.0" label="Maximal difference of the apex in retention time" help=""/>
-    <param name="max_intensity_cutoff" argument="-max_intensity_cutoff" type="float" optional="false" value="0.0" label="Maximal intensity to be added to a spectrum" help=""/>
-    <param name="add_precursor" argument="-add_precursor" type="float" optional="false" value="0.0" label="Add a precursor mass" help=""/>
+    <param argument="-in" type="data" format="consensusxml" optional="false" label="Mass traces" help=" select consensusxml data sets(s)"/>
+    <param argument="-min_pearson_correlation" type="float" optional="false" value="0.7" label="Minimal pearson correlation score" help=""/>
+    <param argument="-min_peak_nr" type="integer" optional="false" value="1" label="Minimal peak nr to output pseudo spectra" help=""/>
+    <param argument="-max_lag" type="integer" optional="false" value="1" label="Maximal lag" help=""/>
+    <param argument="-max_rt_apex_difference" type="float" optional="false" value="5.0" label="Maximal difference of the apex in retention time" help=""/>
+    <param argument="-max_intensity_cutoff" type="float" optional="false" value="0.0" label="Maximal intensity to be added to a spectrum" help=""/>
+    <param argument="-add_precursor" type="float" optional="false" value="0.0" label="Add a precursor mass" help=""/>
     <expand macro="adv_opts_macro">
-      <param name="force" argument="-force" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Overrides tool-specific checks" help=""/>
-      <param name="test" argument="-test" type="hidden" optional="false" value="False" label="Enables the test mode (needed for internal use only)" help="">
+      <param argument="-force" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Overrides tool-specific checks" help=""/>
+      <param argument="-test" type="hidden" optional="false" value="False" label="Enables the test mode (needed for internal use only)" help="">
```

Comment:
test and force should probably not be exposed to the user?

Reply:
You are right with `force`. `test` is hidden and needed for the tests passing.

Reply:
Actually it seems that `-force` is needed https://github.com/OpenMS/OpenMS/issues/5921